### DISCUSSION TITLE
Is there a way to set that All games that I added to Retroarch will be added to Gamehub automatically and the other way round?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#474] [Q&A] [answered]
That would be cool  so I don't have to add all my Roms manually and if I change which core a game uses it's also changed for Retroarch.

**Comment by tkashkin** [comment, marked as answer]
Currently there's no automatic import of RetroArch games, maybe it will be implemented in the future.

**Comment by ghost** [comment]
Is there already a feature request for that?

**Comment by ghost** [comment]
Didn't find one,I created [one](https://github.com/tkashkin/GameHub/issues/478)
